readme.md

```diff
@@ -291,6 +291,11 @@ You can functionally compose your store any way you like.
 ```jsx
 // Log every time state is changed
 const log = (config) => (set, get, api) =>
+  api.setState = (...args) => {
+    console.log('  applying', args)
+    api.setState(...args)
```

Comment:
Infinite loop. 😄

Reply:
Oops, I'll fix that. I ran into that with zundo long ago.

Reply:
but, as i said, my suggestion is remove this section in readme.